src/plugins/dbus/README.md

```diff
@@ -20,7 +20,7 @@ with Elektra.
 
 - `libdbus-1-dev`
 
-## Dbus
+## Dbus ##
```

Comment:
No, I added the closing hashes just for unification, because we use them at most places. Spaces after the leading hashes are necessary though, so `##Something##` would not work, only `## Something ##`.